docs/backup-restore.md

```diff
@@ -0,0 +1,107 @@
+# Scalar DL Backup Creation and Restoration
+
+Since Scalar DL provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
+This guide shows you how to create and restore transactionally-consistent Scalar DL backups.
+
+## How to create transactionally consistent backup 
+
+Scalar DL service is integrated with the admin interface, which allows you to pause the Scalar DL services using the scalar-admin client tool.
+You can use the scalar-admin client tool as a docker container or  fat jar, you can find scalar-admin-<version>-all.jar in the [releases](https://github.com/scalar-labs/scalar-admin/releases).
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
+* You must be able to identify a common restore point for the Ledger and Auditor If you use Auditor.
```

Comment:
I have a suggestion to change this sentence a bit if its okay.
* You must identify a common restore point for the Ledger and Auditor if you also use Auditor.

Reply:
Thank you for your review comments.
Document updated 6c3bee7